pymmh3.py

```diff
@@ -73,14 +72,345 @@ def fmix( h ):
     if tail_size >= 1:
         k1 ^= key[ tail_index + 0 ]
     
-    if tail_size != 0:
+    if tail_size > 0:
         k1  = ( k1 * c1 ) & 0xFFFFFFFF
-        k1  = ( k1 << 15 | k1 >> 17 ) & 0xFFFFFFFF # _ROTL32
+        k1  = ( k1 << 15 | k1 >> 17 ) & 0xFFFFFFFF # inlined ROTL32
         k1  = ( k1 * c2 ) & 0xFFFFFFFF
         h1 ^= k1
 
+    #finalization
     return fmix( h1 ^ length )
 
+def hash128( key, seed = 0x0, x64arch = True ):
```

Comment:
I really can't see why you would want the x64arch flag? That will force your users to check the current arch. Is there any performance-reason to have both implementations?
If there is a performance difference for this code, why not just do a selection of the function to use at module load and then always use the correct one for the current platform?